Views/QRCodeImage.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  QRCodeImage.swift
+//  SwiftUILibrary
+//
+//  Created by Mathijs Bernson on 04/07/2023.
+//
+
+import SwiftUI
+
+struct QRCodeImage: View {
+    let data: Data?
+
+    init(url: URL) {
+        data = url.absoluteString.data(using: .utf8)
+    }
+
+    init(string: String) {
+        data = string.data(using: .utf8)
+    }
+
+    init(data: Data) {
+        self.data = data
+    }
+
+    var body: some View {
+        ImageView(uiImage: generateQRCode(from: data))
+            .aspectRatio(contentMode: .fit)
+
+        // Image(uiImage: generateQRCode(from: data) ?? UIImage())
+        //     .resizable()
+        //     .scaledToFit()
```

Comment:
For some reason, this doesn't work with a SwiftUI Image, but it works when I use a wrapped UIImageView... No idea why.